strapi/package.json

```diff
@@ -17,12 +17,12 @@
   },
   "devDependencies": {},
   "dependencies": {
-    "@strapi/plugin-graphql": "4.25.20",
-    "@strapi/plugin-i18n": "4.25.20",
-    "@strapi/plugin-upload": "4.25.20",
-    "@strapi/plugin-users-permissions": "4.25.20",
+    "@strapi/plugin-graphql": "4.25.22",
+    "@strapi/plugin-i18n": "4.25.22",
+    "@strapi/plugin-upload": "4.25.22",
+    "@strapi/plugin-users-permissions": "4.25.22",
     "@strapi/provider-email-mailgun": "^5.6.0",
```

Comment:
Please change also `@strapi/provider-email-mailgun` to `"4.25.22"`. I wass installed as v5 accidentally, we discussed it already with @jankaBehal.